packages/app/README.md

```diff
@@ -53,19 +69,34 @@ const config: EnvironmentConfig = {
     },
     // next network has been just added
     {
-      apiUrl: "http://localhost:3030",
-      verificationApiUrl: "https://zksync2-testnet-explorer.zksync.dev",
-      hostnames: ["localhost"],
-      icon: "/images/icons/zksync-arrows.svg",
-      l2ChainId: 270,
-      l2NetworkName: "Local Hyperchain",
-      maintenance: false,
-      name: "local-hyperchain",
-      published: true,
-      rpcUrl: "http://localhost:3070",
+      "groupId": "era",
+      "apiUrl": "https://block-explorer-api.sepolia.zksync.dev",
+      "verificationApiUrl": "https://explorer.sepolia.era.zksync.dev",
+      "bridgeUrl": "https://portal.zksync.io/bridge/?network=sepolia",
+      "hostnames": [
+        "https://sepolia.staging-scan-v2.zksync.dev"
```

Comment:
?

Reply:
That's what's in the [staging.config](https://github.com/matter-labs/block-explorer/blob/3abc188c54c9db8cf7385a4e4cc7899233b9767e/packages/app/src/configs/staging.config.json#L4) so I kept it the same for consistency. 

I can update the staging.config so I can update this

Reply:
Changed to link to production.config